docs/schema-support.md

```diff
@@ -0,0 +1,72 @@
+# Schema support
```

Comment:
I'd clarify that this is specifically for CloudFormation templates somewhere near the start of the document.
We have schemas for a couple of other things too but they're all sourced from different places. 

Could also rename the doc: `cfn-schema-support`